resources/metadata/whitesource.yaml

```diff
@@ -0,0 +1,523 @@
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: 15
+    - name: reporting
+      type: bool
+      description: Whether assessment is being done at all, defaults to `true`.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: whitesource/serviceUrl
+      type: string
+      description: URL to the WhiteSource server API used for communication, defaults to `https://saas.whitesourcesoftware.com/api`.
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: https://saas.whitesourcesoftware.com/api
+    - name: buildDescriptorExcludeList
+      type: string
+      description: List of build descriptors and therefore modules to exclude from the scan and assessment activities.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: []
+    - name: whitesource/orgToken
+      type: string
+      description: WhiteSource token identifying your organization.
+      mandatory: true
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: licensingVulnerabilities
+      type: bool
+      description: Whether license compliance is considered and reported as part of the assessment.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: agentFileName
+      type: string
+      description: Locally used name for the Unified Agent jar file after download.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: wss-unified-agent.jar
+    - name: emailAddressesOfInitialProductAdmins
+      type: string
+      description: The list of email addresses to assign as product admins for newly created WhiteSource products.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: []
+    - name: whitesource/productVersion
+      type: string
+      description: Version of the WhiteSource product to be created and used for results aggregation, usually determined automatically.
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: whitesource/jreDownloadUrl
+      type: string
+      description: URL used for downloading the Java Runtime Environment (JRE) required to run the WhiteSource Unified Agent.
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: whitesource/productName
+      type: string
+      description: Name of the WhiteSource product to be created and used for results aggregation.
+      mandatory: true
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: verbose
```

Comment:
Please remove, `verbose` is available as a general parameter for all commands and need not to be mentioned.